include/bluetooth/mesh/scheduler_srv.rst

```diff
@@ -0,0 +1,222 @@
+  +-----------+----------------------------------------------------------+
+  | 0x00�0x3B | Second of the minute (00 to 59)                          |
+  +-----------+----------------------------------------------------------+
+  | 0x3C      | Any second of the minute                                 |
+  +-----------+----------------------------------------------------------+
+  | 0x3D      | Every 15 seconds (second modulo 15 is 0) (0, 15, 30, 45) |
+  +-----------+----------------------------------------------------------+
+  | 0x3E      | Every 20 seconds (second modulo 20 is 0) (0, 20, 40)     |
+  +-----------+----------------------------------------------------------+
+  | 0x3F      | Once a minute (at a random second)                       |
+  +-----------+----------------------------------------------------------+
+
+* DayOfWeek: Bitmask of the days of the week when the scheduled event is enabled
+
+  +-----+-------------------------+
+  | Bit | Description             |
+  +-----+-------------------------+
+  | 0   | Scheduled on Mondays    |
+  +-----+-------------------------+
+  | 1   | Scheduled on Tuesdays   |
+  +-----+-------------------------+
+  | 2   | Scheduled on Wednesdays |
+  +-----+-------------------------+
+  | 3   | Scheduled on Thursdays  |
+  +-----+-------------------------+
+  | 4   | Scheduled on Fridays    |
+  +-----+-------------------------+
+  | 5   | Scheduled on Saturdays  |
+  +-----+-------------------------+
+  | 6   | Scheduled on Sundays    |
+  +-----+-------------------------+
+
+* Action: Action to be executed for a scheduled event
+
+  +------------------+-------------------------+
+  | Value            | Description             |
+  +------------------+-------------------------+
+  | 0x0              | Turn Off                |
+  +------------------+-------------------------+
+  | 0x1              | Turn On                 |
+  +------------------+-------------------------+
+  | 0x2              | Scene Recall            |
+  +------------------+-------------------------+
+  | 0xF              | No action               |
+  +------------------+-------------------------+
+  | All other values | Reserved for Future Use |
+  +------------------+-------------------------+
+
+* Transition Time: Transition time for the action
+
+  * Step count: 6 bits (range `0x00` to `0x3e`)
+  * Step resolution: 2 bits
+
+* Scene Number: Scene number to be used for the action
+
+  +------------------+--------------+
+  | Value            | Description  |
+  +------------------+--------------+
+  | 0x0000           | No scene     |
+  +------------------+--------------+
+  | All other values | Scene number |
+  +------------------+--------------+
+
+Extended models
+===============
+
+The Scheduler Server extends the :ref:`bt_mesh_scene_srv_readme`.
+In addition, the Scheduler Setup Server extends the Scene Setup Server (see the :ref:`bt_mesh_scene_srv_readme` documentation) and the Generic Power OnOff Setup Server.
```

Comment:
Not mentioning the Setup Server in the Generic Power OnOff Server doc... Should we?

Reply:
I'll add the reference on Power OnOff server doc here to clarify

Reply:
Sure, I am the one who got confused here, looking at the OnOff Server LOL